### DISCUSSION TITLE
UX Changes : Bot start and stop

### DISCUSSION THREAD

**Discussion by rvinothrajendran** [microsoft/BotFramework-Composer#7183] [Design]
As user its not easy way to stop the bot , every time go to "Open start bot panel" and stop bot 

## Describe the solution you'd like
better add the button to stop the bot on the panel

![image](https://user-images.githubusercontent.com/16264167/115028515-3df2cb80-9ec5-11eb-8012-8882b1472bd3.png)

**Comment by benbrown** [comment]
@mewa1024 What do you think about this suggestion?

**Comment by mewa1024** [comment]
I agree, but adding @sangwoohaan since I think there was some history to how the stop mechanism ended up in the drawer. Sang, thoughts?

**Reply by sangwoohaan** [reply]
The thought here is to have either restart or start/stop, not both of them due to the complication of interactions and lack of proper user scenarios to provide all of them. Initially, there was only start/stop replacing restart for both all bot and individual bot control, then it was changed to the current model in which you have a one-click restart for all bots, and start/stop (more granular control) for individual bots. We can definitely dig into this and start thinking about enabling both, but there should be solid user scenarios before we make this control much more complex.

**Comment by mewa1024** [comment]
We will add this to design backlog. Tracking in #5135
